neukinnis/spec/integration/jeff-aboutus.spec.js

```diff
@@ -0,0 +1,41 @@
+//jeff-aboutus.spec.js
+
+/* global casper */
+
+casper.test.begin('About us page test', 3,
```

Comment:
I found this a little difficult to read at first because the indentation doesn't seem to line up with the curly braces.  Here's two issues I noticed

---

Instead of
```
casper.test.begin('About us page test', 3,
  function suite(test){ // open brace here
    // ...
}); // close brace out here
```
You might use
```
casper.test.begin('About us page test', 3, function suite(test){
  // ...
});
```

---

Also it looks like some code got outdented here by accident
```
    casper.start('http://localhost/neukinnis/client/index.html', function(){
    test.assertTitle('Neukinnis', 'Finds the main page with title: Neukinnis');
    test.assertExists('a[href="aboutus.html"]', 'Checks existence link to about us page');
  });
```

I think this was supposed to be more like this
```
    casper.start('http://localhost/neukinnis/client/index.html', function(){
      test.assertTitle('Neukinnis', 'Finds the main page with title: Neukinnis');
      test.assertExists('a[href="aboutus.html"]', 'Checks existence link to about us page');
    });
```